src/modules/nats.rs

```diff
@@ -0,0 +1,96 @@
+use super::{Context, Module, ModuleConfig};
+use serde_json as json;
+
+use crate::configs::nats::NatsConfig;
+use crate::formatter::StringFormatter;
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("nats");
+    let config = NatsConfig::try_load(module.config);
+
+    if config.disabled {
+        return None;
+    };
+
+    let ctx_str = context
+        .exec_cmd("nats", &["context", "info", "--json"])?
+        .stdout;
+    let nats_context: json::Value = json::from_str(&ctx_str).ok()?;
+    let name = nats_context.get("name")?.as_str()?;
```

Comment:
I feel like failure to parse the JSON should be logged here.
Additionally, I would prefer to move the `nats_context.get("name")` into the `name =>` `match`, but you can skip that if that causes any issues.

Reply:
yeah I can do that. I think I looked at another module to see how others did it and assumed the error bubbled up.